src/App.tsx

```diff
@@ -1,16 +1,58 @@
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import Logo from "./logo.png";
+import { ValidateEmail, ValidatePassword } from "./utils/validate";
 
 function App() {
+  const [email, setEmail] = useState(" ");
+  const [password, setPassword] = useState(" ");
+  const [errorEmail, setErrorEmail] = useState("");
+  const [errorPassword, setErrorPassword] = useState("");
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    ValidateEmail(email) ? setErrorEmail("E-mail Inválido") : setErrorEmail("");
+
+    ValidatePassword(password)
+      ? setErrorPassword(
+          "Password Inválido Lembre-se: Mínimo 7 caracteres, deve conter, ao menos ,uma letra e um número."
+        )
+      : setErrorPassword("");
+    e.preventDefault();
+  };
+
+  const handleChangeEmail = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value);
+  };
+
+  const handleChangePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+
   return (
     <div className="App">
       <header className="App-header">
         <p>Bem-vindo à Taqtile</p>
         <img src={Logo} width="100px" height="100px" />
-        <input placeholder="E-mail" />
-        <input placeholder="Senha" />
-        <button>Entrar</button>
+        <form
+          onSubmit={(e) => {
+            handleSubmit(e);
+          }}
```

Comment:
```suggestion
          onSubmit={handleSubmit}
```

Atalhozinho do javascript pra quando vc quer chamar uma função apenas repassando os parâmetros recebidos.